src/Compilers/VisualBasic/Test/Syntax/Scanner/ScannerTests.vb

```diff
@@ -508,9 +508,39 @@ Public Class ScannerTests
         Assert.Equal(" _ ", tk.ToFullString())
 
         tk = ScanOnce(" _'", startStatement:=True)
-        Assert.Equal(SyntaxKind.BadToken, tk.Kind)
+        Assert.Equal(SyntaxKind.EndOfFileToken, tk.Kind)
         Assert.Equal(" _'", tk.ToFullString())
-        Assert.Equal(30999, tk.Errors.First().Code)
+        Assert.True(tk.LeadingTrivia(0).Kind = SyntaxKind.WhitespaceTrivia)
+        Assert.True(tk.LeadingTrivia(1).Kind = SyntaxKind.LineContinuationTrivia)
+        Assert.True(tk.LeadingTrivia(2).Kind = SyntaxKind.CommentTrivia)
+        Assert.Equal(37306, tk.Errors.First().Code)
+
+        ' PROTOTYPE LanguageVersion.VisualBasic15_5 should be LanguageVersion.VisualBasic16
+        tk = ScanOnce(" _'", LanguageVersion.VisualBasic15_5)
+        Assert.Equal(SyntaxKind.EndOfFileToken, tk.Kind)
+        Assert.Equal(" _'", tk.ToFullString())
+        Assert.True(tk.LeadingTrivia(0).Kind = SyntaxKind.WhitespaceTrivia)
+        Assert.True(tk.LeadingTrivia(1).Kind = SyntaxKind.LineContinuationTrivia)
+        Assert.True(tk.Errors.Count = 0)
```

Comment:
>Assert.True(tk.Errors.Count = 0) [](start = 8, length = 32)

Shouldn't we also have a CommentTrivia?

Reply:
yes, I need to put a count Assert on all of them